index.html

```diff
@@ -1026,8 +1208,13 @@ <h3>Role Mapping Table</h3>
 								</td>
 								<td><p>Expose <code>ROLE_LANDMARK</code> and object attribute
 											<code>xml-roles:doc-pagebreak</code>. </p></td>
```

Comment:
@clapierre can you verify if this is the issue you had and open a new issue to correct if so?

Otherwise, please give the updated PR a review and approve it if you're good with the changes now.

Reply:
Yes I filed #16 as a separate issue for the page break/preface issue, this PR looks good and I will approve it :)